### PR TITLE
Gio/section component

### PR DESCRIPTION
**Pull request by digmel**
section component added

### CODE REVIEW

File: src/components/section/Section.style.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import {StyleSheet} from 'react-native';
+
+export const styles = StyleSheet.create({
+  container: {
+    flexGrow: 1,
```
Comment (lasharela) [inline src/components/section/Section.style.ts:5]: not needed.

File: src/components/section/Section.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+import {size} from '@components';
+
+export const Section: FC<TSectionProps> = ({
+  containerStyle,
+  isFullWidth = false,
+  children,
+  ...props
+}) => {
+  const fullWidthStyle: ViewStyle = isFullWidth
+    ? {marginHorizontal: -size.xs}
```
Comment (lasharela) [inline src/components/section/Section.tsx:14]: isFullWidth && {marginHorizontal: -size.xs}

File: src/components/section/Section.tsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import React, {FC} from 'react';
+import {View, ViewStyle} from 'react-native';
+import {styles} from './Section.style';
+import {TSectionProps} from './Section.type';
+import {size} from '@components';
+
+export const Section: FC<TSectionProps> = ({
+  containerStyle,
```
Comment (lasharela) [inline src/components/section/Section.tsx:8]: - bottom - by default 16
- top - optional, 0

File: src/components/section/sectionStories.tsx [hunk 4]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import {storiesOf} from '@storybook/react-native';
+import {color, Screen} from '@components';
+import {Section} from './Section';
+import {View} from 'react-native';
+
+export const sectionStories = storiesOf('Section', module);
+
+sectionStories.add('Regular', () => (
+  <Screen>
```
Comment (lasharela) [inline src/components/section/sectionStories.tsx:10]: 1 story is enough

**Review by lasharela — APPROVED** [review]

File: src/components/section/Section.style.ts (hunk 1 above)
Comment (digmel) [reply src/components/section/Section.style.ts:5]: done

File: src/components/section/Section.tsx (hunk 2 above)
Comment (digmel) [reply src/components/section/Section.tsx:14]: done

File: src/components/section/Section.tsx (hunk 3 above)
Comment (digmel) [reply src/components/section/Section.tsx:8]: got it

File: src/components/section/sectionStories.tsx (hunk 4 above)
Comment (digmel) [reply src/components/section/sectionStories.tsx:10]: done

**Pull request merged**